### DISCUSSION TITLE
Behaviour whith logging -> stdout = false

### DISCUSSION THREAD

**Discussion by maticomba** [postalserver/postal#2166] [Help with using Postal] [answered]
Hello! I couldn't find anywhere what is the behaviour when setting to false stdout variable in the logging section inside postal.yml. I guess some file will be created, but where?

My goal is to bind that file to use fail2ban to protect smtp server fom bruteforce attacks.

Thanks in advance!

**Comment by willpower232** [comment, marked as answer]
You can see here that a file is written to `log_root`

https://github.com/postalserver/postal/blob/152633adfe861699fd7d762690d0c521c31d2dbf/lib/postal/config.rb#L100-L107

which is either `app/log` or a folder you specify in the config file

https://github.com/postalserver/postal/blob/152633adfe861699fd7d762690d0c521c31d2dbf/lib/postal/config.rb#L48-L52

I'd suggest mounting a directory that you can bind to all containers and then accessing the file in there so it can be managed cleanly by Postal without potentially upsetting any direct docker bind

**Reply by maticomba** [reply]
Thank you Will, this is exactly what I needed!

I made a small tutorial on how to I made it work with Ubuntu Server 22.04.1 and Postal v2.11.2.

https://github.com/maticomba/postal-smtp-fail2ban
